### ISSUE TITLE
Allow for chromedriver-filepath to be the end chromedriver binary or a zip

### ISSUE THREAD

**Issue by kpaxton** [giggio/node-chromedriver#139]
I'm trying to use this in a closed network that does not have internet access to download the binaries from the cdn. I do however have the npm package ```chrome-driver-standalone``` installed. This has the extracted binaries available in its folders. I have tried to set the ```chromedriver-filepath``` to point to it but your install script seems to expect a zip file. Is there another config parameter that I can set to tell it not to try and extract a binary?

How can I get this to work?

**Comment by kpaxton** [comment]
As a workaround I installed the chrome-driver-standalone globally, copied and renamed the binary I needed to ```chromedriver```, put it in a zip named ```chromedriver_linux32.zip``` and set the ```chromedriver-filepath``` to point to it. 

My suggestion is to either create a new config parameter ```chromedriver-binarypath``` which will point to an already extracted binary. OR your install script should be able to handle that the outcome of the ```filepath``` could be a ```.zip``` or the binary itself.
